src/App.js

```diff
@@ -1,5 +1,6 @@
 import './App.css';
```

Comment:
Usually we end React files with .jsx. Although it's important to note that React Typescript files end in .tsx

Reply:
Will try typescript in a different pull request